pkg/builder/intg_test_context.go

```diff
@@ -0,0 +1,273 @@
+// Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package builder
+
+import (
+	"context"
+	"fmt"
+	"path/filepath"
+
+	"github.com/go-logr/logr"
+	"github.com/google/uuid"
+
+	// nolint
+	. "github.com/onsi/ginkgo"
+
+	// nolint
+	. "github.com/onsi/gomega"
```

Comment:
nit: remove the `nolint` directive
```suggestion
	. "github.com/onsi/ginkgo"
	. "github.com/onsi/gomega"
```